amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -406,21 +406,22 @@ type UserStakes {
 type StakerRewards {
   address: String!
   rewards: MotionStakeValues!
+  isClaimed: Boolean!
 }
 
 type MotionData {
-  motionId: String!
+  motionId: String! # to ensure uniqueness, we format as: chainId-votingRepExtnAddress_nativeMotionId
+  nativeMotionId: String!
   usersStakes: [UserStakes!]!
   stakerRewards: [StakerRewards!]!
   motionStakes: MotionStakes!
   remainingStakes: [String!]! # tuple [nayRemaining, yayRemaining]
   userMinStake: String!
   requiredStake: String!
-  # For calculating user's max stake in client
-  rootHash: String!
-  # native domain id
-  motionDomainId: String!
+  rootHash: String! # For calculating user's max stake in client
+  motionDomainId: String! # native domain id
   isFinalized: Boolean!
+  createdBy: String! # voting rep extn address. Useful to check if we're viewing a "read-only" motion
```

Comment:
I anticipate this being useful when it comes to viewing motions that were created by a now-deleted voting reputation client. 

It would allow for a very simple check in the front-end for whether the motion is "current", and if not, render it "read-only" - i.e. can't be interacted with. It's essentially freezing the motion at the state it was when its voting client was uninstalled.